.vib/grafana/cypress/cypress/integration/grafana_test_suite.js

```diff
@@ -0,0 +1,146 @@
+/// <reference types="cypress" />
+import {
+  deleteDashboard,
+  deleteAPIKey,
+  clickDeleteButton,
+  verifySuccesOfAction
+} from './utils'
+
+it('allows user to log out', () => {
+  cy.login();
+  cy.visit('/logout');
+  cy.get('.login-content-box').should('exist');
+})
+
+it('allows creating a dashboard with a panel', () => {
+  const CLOSE_DIALOGUE_BUTTON = 'button[aria-label="Close dialogue"]';
+
+  cy.login();
+  cy.visit('dashboard/new');
+  cy.get('button').contains('Add a new panel')
+    .should('be.visible').click();
+  cy.get('[value="Panel Title"]').clear().type(Cypress.env('panel_title'));
+  cy.get('textarea#description-text-area')
+    .should('exist').type(Cypress.env('panel_description'));
+  cy.get('button').contains('Apply')
+    .should('be.visible').click();
+  cy.get('.panel-title').contains(Cypress.env('panel_title'));
+  cy.get('[aria-label="Save dashboard"]').click();
+  cy.get(CLOSE_DIALOGUE_BUTTON)
+    .should('be.visible');
+  cy.get('[name="title"]').clear().type(Cypress.env('dashboard_title'));
+  cy.get('[type="submit"]')
+    .should('be.visible').click();
+  verifySuccesOfAction();
+  deleteDashboard();
+})
+
+it('checks if it is possible to upload a dashboard as JSON file', () => {
+  const IMPORT_DASHBOARD_BUTTON = '[data-testid="data-testid-import-dashboard-submit"]';
+  
+  cy.login();
+  cy.visit("dashboard/import");
+  cy.get('label').contains('Upload JSON').click();
+  cy.get('input[type=file]').selectFile('cypress/fixtures/test-dashboard.json',
+  {force:true});
+  cy.get(IMPORT_DASHBOARD_BUTTON).click();
+  cy.visit("dashboards");
+  cy.get('div').contains('New test dashboard')
```

Comment:
As the `test-dashboard.json` is a JSON file, could it be possible to parse it as a fixture and use `fixture.title` instead of hardcoding the test value?

Reply:
Before I read your comment I've applied the following solution to this problem:
` cy.get('[data-testid="data-testid-import-dashboard-title"]').clear().type(${DASHBOARD_TITLE}.${random});`
This will replace the hardcoded title with a randomised one.
I've tested this by executing the test suite many times in a row, I confirm this works.
I think both your approach and mine and valid and achieve the same result.